modules/lti/src/OpencastRest.ts

```diff
@@ -224,7 +243,8 @@ export async function searchEpisode(
                         url: attachment.url
                     })),
                 tracks: parseTracksFromResult(result)
-            }
+            },
+            isLive: result.mediapackage.media.track !== undefined ? result.mediapackage.media.track.isLive : false
```

Comment:
I haven't actually tested this, but I'm not sure if this will always work. It's quite annoying, but `track` may be a JS object *or* an array. That's why there are all the weird checks above. I think code will not work if there are multiple tracks (i.E. `track` is an array).